apps/myaccount/src/configs/ui.ts

```diff
@@ -209,62 +212,62 @@ export const getEmptyPlaceholderIllustrations = (): {
     search: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
     newList: React.FunctionComponent<React.SVGProps<SVGElement>>;
  } => {
- 
+
     return {
         accessDeniedError: ForbiddenIcon,
         emptyList: BoxIcon,
         genericError: CloseIcon,
         loginError: ForbiddenIcon,
-        pageNotFound: BlockedMagnifierIcon,
         newList: EmptyListResultsIllustration,
+        pageNotFound: BlockedMagnifierIcon,
         search: EmptySearchResultsIllustration
     };
 };
- 
+
 export const getAppIconBackgrounds = (): {
     orange: string;
  } => {
- 
+
     return {
         orange: OrangeAppIconBackground
     };
 };
- 
+
 export const getThirdPartyLogos = (): {
     gravatar: string;
  } => {
- 
+
     return {
         gravatar: GravatarLogo
     };
 };
- 
+
 export const AlertIcons = (): {
     error: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
     info: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
     success: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
     warning: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
  } => {
- 
+
     return {
         error: ErrorIcon,
         info: InfoIcon,
         success: SuccessIcon,
         warning: WarningIcon
     };
 };
- 
-export const AppSwitcherIcons = (): Record<string, any> => { 
+
+export const AppSwitcherIcons = (): Record<string, any> => {
```

Comment:
Shall we try this type for better IntelliSense.
```suggestion
export const AppSwitcherIcons = (): {
    console: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
    myAccount: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
 } => {
```